templates/digital-asset-template/frontend/hooks/useGetCollections.tsx

```diff
@@ -0,0 +1,93 @@
+import { aptosClient } from "@/utils/aptosClient";
+import { AccountAddress } from "@aptos-labs/ts-sdk";
+import { useState, useEffect } from "react";
+
+/**
+ * A react hook to get all collections under the current contract.
+ *
+ * This call can be pretty expensive when fetching a big number of collections,
+ * therefore it is not recommended to use it in production
+ *
+ */
+export function useGetCollections() {
+  const [fas, setFAs] = useState<any[]>([]);
```

Comment:
Any possibility to narrow this down from `any` to something more specific?

Reply:
oops, good catch. that should be renamed and typed